src/styles/index.scss

```diff
@@ -1,3 +1,116 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: solid 1px #000;
```

Comment:
color should not be black, check mockup and pick proper one.

Reply:
done